src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/hmi_command_factory.cc

```diff
@@ -623,16 +606,6 @@ CommandCreator& HMICommandFactory::get_creator_factory(
                  ? factory.GetCreator<commands::UIIsReadyRequest>()
                  : factory.GetCreator<commands::UIIsReadyResponse>();
     }
-    case hmi_apis::FunctionID::VehicleInfo_IsReady: {
-      return hmi_apis::messageType::request == message_type
-                 ? factory.GetCreator<commands::VIIsReadyRequest>()
-                 : factory.GetCreator<commands::VIIsReadyResponse>();
-    }
-    case hmi_apis::FunctionID::VehicleInfo_ReadDID: {
-      return hmi_apis::messageType::request == message_type
-                 ? factory.GetCreator<commands::VIReadDIDRequest>()
-                 : factory.GetCreator<commands::VIReadDIDResponse>();
-    }
 #ifdef HMI_DBUS_API
     case hmi_apis::FunctionID::VehicleInfo_GetGpsData: {
       return hmi_apis::messageType::request == message_type
```

Comment:
why VehicleInfo_GetClusterModeStatus is not moved to the new plugin?

Reply:
@yang1070 VehicleInfo_GetClusterModeStatus is applicable only for HMI_DBUS_API that is not supported by SDL right now and as I know will be removed within 5.0 ( same version as for this changes)